underlay/src/main/java/bio/terra/tanagra/utils/RetryUtils.java

```diff
@@ -164,10 +164,10 @@ public static <T, E extends Exception> T pollWithRetries(
           // the exception is not retryable: re-throw
           throw ex;
         } else {
-          // keep track of the last retryable exception so we can re-throw it in case of a timeout
+          // keep track of the last retryable exception, so we can re-throw it in case of a timeout
           lastRetryableException = ex;
         }
-        LOGGER.debug("Caught retryable exception: {}", ex);
+        LOGGER.debug("Caught retryable exception {}", ex.getMessage());
```

Comment:
I don't think this does the same thing. We want to log the exception (including cause, etc.) not just the message.

Reply:
It doesn't - This was a warning since debug expected (string, throwable). 
fixing it to log the ex by removing the placeholder

Reply:
Oh I see, my mistake. Yes, your fix is what I intended. Thanks for taking care of it!